core/src/main/java/org/yggard/brokkgui/behavior/GuiTextfieldBehavior.java

```diff
@@ -129,4 +147,95 @@ public void setCursorPosition(final int cursorPosition)
             this.getModel().setCursorPosition(cursorPosition);
         }
     }
+    
+    private static final Pattern alphaNumRegex = Pattern.compile("[a-zA-Z0-9]");
```

Comment:
Should be at the top of the file with package-level access for subTypes of TextField.
Also named in FULL_CASE.